src/midi/midi_fluidsynth.cpp

```diff
@@ -465,6 +482,15 @@ bool MidiHandlerFluidsynth::Open([[maybe_unused]] const char *conf)
 		mixer_channel->SetLowPassFilter(FilterState::Off);
 	}
 
+	// Double the baseline PCM prebuffer because MIDI is demanding
+	const auto render_ahead_ms = MIXER_GetPreBufferMs() * 2;
+
+	static constexpr auto max_frames_per_ms = 48;
+	frame_fifo.Resize(render_ahead_ms * max_frames_per_ms);
```

Comment:
Shouldn't this be dependent on the actual sample rate instead of assuming 48k? (48 = 48000 / 1000 in this case)

Reply:
Used actual mixer rate instead of the maximum.